app/representers/api/v1/exercises/representer.rb

```diff
@@ -67,7 +67,7 @@ def self.cache_key_types_for(represented, options = {})
     end
 
     def self.cache_key_for(represented, type)
-      "#{represented.cache_key}/#{type}"
+      "#{represented.cache_key}/##{represented.cache_version}/#{type}"
```

Comment:
Hmm, just realized I doubled the ## there, so this might not expand correctly. (tested, not an issue, just not very neat)
To use the version at fetch time, I'd need to convert all the `cache_key_for*` style methods to return 2-value arrays, and use those both when accessing the actual cache, I guess.

Reply:
Ah yes the double ## will invalidate all the caches. But I think the change from cache_key to cache_version might too, since Rails used a dash before to join them. If you want to avoid clearing the existing cache, there is also a `cache_key_with_version` method now.